src/Symfony/Bridge/Twig/Extension/HtmlSanitizerExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Psr\Container\ContainerInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Titouan Galopin <tgalopin@gmail.com>
+ */
+final class HtmlSanitizerExtension extends AbstractExtension
+{
+    public function __construct(
+        private ContainerInterface $sanitizers,
+        string $defaultName = 'default',
+    ) {
+    }
+
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('sanitize_html', $this->sanitize(...), ['is_safe' => ['html']]),
```

Comment:
IIRC, Twig does not yet support the first-class callable notation to compile extension methods to the optimized code that does not require loading all filters at runtime.

Reply:
We use this notation for all filters in 6.1.

> compile extension methods to the optimized code

I had a look at twig's source and I'm not sure what you mean. Do you have a pointer to what you're thinking of?

Reply:
See `Twig\Node\Expression\CallExpression::compileCallable`

Reply:
Thanks, should be fixed by https://github.com/twigphp/Twig/pull/3702